### ISSUE TITLE
What happens if someone joins answer phase after end time?

### ISSUE THREAD

**Issue by brandonchinn178** [brandonchinn178/scattergories#51]
cc @matthew-chinn

**Comment by matthew-chinn** [comment]
we could make a refresh button that just returns all of the current state?

**Comment by brandon-leapyear** [comment]
:sparkles: _**This is an old work account. Please reference @brandonchinn178 for all future communication**_ :sparkles:


---

yeah, or maybe after answers are locked, go to the validation page with maybe a loading message and a "force continue" button?

**Comment by matthew-chinn** [comment]
the refresh button would be available whenever

**Comment by matthew-chinn** [comment]
for moving to validation step, can the server just end the answer page automatically ~3 seconds after the end time? little buffer for any different local clocks

**Comment by brandonchinn178** [comment]
I can make it 10 seconds for now?

**Comment by matthew-chinn** [comment]
i think 3-5 range could be better cause 10 seconds of just sitting there feels like forever when you're just waiting staring at a webpage

**Comment by matthew-chinn** [comment]
This issue can be for the server to auto-end the answer round after the time limit is done, I'll make another issue for the "refresh all" request

**Comment by brandonchinn178** [comment]
yeah, working on it here: https://github.com/brandonchinn178/categories-with-friends/compare/timeout

currently not working, if a player disconnected completely

**Comment by brandonchinn178** [comment]
@matthew-chinn does #56 fix this?
